postgres/datadog_checks/postgres/version_utils.py

```diff
@@ -18,21 +18,38 @@ def get_version(db):
     cursor = db.cursor()
     cursor.execute('SHOW SERVER_VERSION;')
     raw_version = cursor.fetchone()[0]
+    version = _parse_version(raw_version)
+    return raw_version, version
```

Comment:
(nit, feel free to ignore)

Returning a tuple for `get_version` seems not very natural. What about:

```

def get_version(db):
    return raw_version

def parse_version(raw_version):
    return version

# caller
    @property
    def version(self):
        if self._version is None:
            raw_version = get_version(self.db)
            self._version = parse_version(raw_version)
            self.set_metadata('version', raw_version)
        return self._version
```